CHANGES.md

```diff
@@ -5,6 +5,7 @@
   - `CloudantClient.schedulerJobs()`,
   - `CloudantClient.schedulerDocs()`,
   - `CloudantClient.schedulerDoc(docId)`.
+- [NEW] Add `Key.ComplexKey.END_KEY`.
```

Comment:
Would it be worth saying what this is actually for here? e.g. appending "for an array high sentinel value".